webapp/src/components/assignee_modal/assignee_modal.jsx

```diff
@@ -0,0 +1,147 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+import AutocompleteSelector from '../user_selector/autocomplete_selector.tsx';
+import Button from '../../widget/buttons/button';
+import IconButton from '../../widget/iconButton/iconButton';
+
+import CompassIcon from '../icons/compassIcons';
+
+const AssigneeModal = ({ visible, close, autocompleteUsers, theme, getAssignee, removeAssignee, removeEditingTodo, changeAssignee, editingTodo }) => {
+    const [assignee, setAssignee] = useState();
+
+    useEffect(() => {
+        function handleKeypress(e) {
+            if (e.key === 'Escape' && visible) {
```

Comment:
If the component gets instantiated as not visible, this won't be executed at any moment, even if visible changes. If the component is instantiated as visible, but then changes, this would be still executed.

Reply:
Should be fixed if we pass visible as a parameter to useEffect.